src/uu/split/src/split.rs

```diff
@@ -7,11 +7,15 @@
 
 // spell-checker:ignore (ToDO) PREFIXaa
 
+#[macro_use]
+extern crate clap;
```

Comment:
I don't think it is needed anymore 
See https://github.com/uutils/coreutils/pull/1673/files

Reply:
that's cool, that's cool... will remove it